### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by cqueern**
noticed (at least in Firefox and Chrome) that it was possible to enter a negative value for minutes using the dial.   

Changed input type from number to range with a minimum of one and max of 100.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div id='head'>
       <div id='intro' class='center'>Enter total workout time in minutes (rests included):</div>
       <div class='center'>
-        <input id='minutes' value='20' type='number'/>
+        <input id='minutes' value='20' type='range' min='1' max='100'/>
```
Comment (diracdeltas) [inline index.html:13]: hi caleb! i like this idea but would like to keep the ability to use fractional values like 0.5 for testing. so maybe min='0' would solve it.

Comment (diracdeltas) [reply index.html:13]: it'd also be nice if it was possible to actually enter/see the time in the input... probably what we want here is a text input field but with some validation

**Comment by diracdeltas** [comment]
actually i have a solution.. one sec! thanks for this

**Comment by diracdeltas** [comment]
https://github.com/diracdeltas/random-training/commit/a8de967f86fa6d0cd7b72bc5a9d87a222d5b2703

File: index.html (hunk 1 above)
Comment (cqueern) [reply index.html:13]: I see what you did. That works great! Was just what I was aiming for. Thanks for all your hard work :)

**Pull request closed without merging**